xterm/index.cgi

```diff
@@ -181,6 +181,18 @@ my $user = $access{'user'};
 if ($user eq "*") {
 	$user = $remote_user;
 	}
+
+# Switch to given user
+if (&webmin_user_is_admin()) {
+	my $username = $in{'user'};
+	if ($username) {
+	my @uinfo = getpwnam($username);
```

Comment:
Better to call error if the provided username doesn't exist

Reply:
Fixed.